packages/driver/src/cypress/error_messages.ts

```diff
@@ -520,6 +520,12 @@ export default {
         docsUrl: `https://on.cypress.io/${_.toLower(obj.cmd)}`,
       }
     },
+    read_timed_out (obj) {
+      return {
+        message: `${cmd('readFile', '"{{file}}"')} timed out.`,
```

Comment:
```suggestion
        message: `${cmd('readFile', '"{{file}}"')} timed out after {{timeout}}ms.`,
```

Reply:
This will result in errors

```
Timed out retrying after 4000ms: `cy.readFile(foo.txt)` timed out after 4000ms.
```

rather than

```
Timed out retrying after 4000ms: `cy.readFile(foo.txt)` timed out.
```

If you think the first one looks better, I'll roll with it, just seems a little repetitive.

Reply:
Ah, I wasn't thinking about it being concatenated to the "Timed out" message, so this would definitely be redundant.